static/js/biomedical/disease/data_processing_utils.test.tsx

```diff
@@ -184,7 +190,7 @@ test("getDiseaseCommonName", () => {
   }[] = [
     {
       data: dataDOID2403 as GraphNodes,
-      wantName: "Hepatitis B",
+      wantName: undefined,
```

Comment:
Should the name still be undefined with this fix ..?

Reply:
So the fix is just changing the `descriptorName` property to `name` property within the disease-symptom association graph because there we're alluding to symptom names instead of disease names. In this case, for the disease name, we need to make a property addition in the config because some diseases have a `commonName` while some have just `name`. I haven't requested the name change in the config yet because I predict we would need to add some properties so it might be nice to submit a CL for all of them together. In the meantime, this fixes the disease-symptom association chart which is nice!